pkg/provision/providers/qemu/node.go

```diff
@@ -84,6 +84,10 @@ func (p *provisioner) createNode(state *vm.State, clusterReq provision.ClusterRe
 	// Talos config
 	cmdline.Append("talos.platform", constants.PlatformMetal)
 
+	// SELinux
+	cmdline.Append("selinux", "1")
+	cmdline.Append("enforcing", "0")
```

Comment:
Maybe we should add options to talosctl for this?

Reply:
we already have `--extra-boot-kernel-args`, but for now let's keep it this way, we can re-consider later